tron/bin/tron_stateless_alert.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3.6
+import argparse
+import logging
+import os
+import time
+
+from tron.config import manager
+from tron.config import schema
+from tron.serialize.runstate.dynamodb_state_store import DynamoDBStateStore
+
+# Default values for arguments
+DEFAULT_WORKING_DIR = '/var/lib/tron/'
+DEFAULT_CONF_PATH = 'config/'
+DEFAULT_ALERT_AFTER = 30
+log = logging.getLogger('tron_stateless_alert')
+
+
+def get_last_run_time(job):
+    '''
+    Get all sorted timestamps, and only count the actions that actually ran
+    '''
+    timestamps = []
+    job_runs = job['runs']
+    for run in job_runs:
+        for action in run['runs']:
+            if action.get('start_time') and action.get('state') != 'scheduled':
+                timestamps.append(action.get('start_time'))
+    timestamps.sort()
+    return timestamps[-1] if timestamps else None
+
+
+def parse_cli():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-w",
+        "--working-dir",
+        default=DEFAULT_WORKING_DIR,
+        help="Working directory for the Tron daemon, default %(default)s",
+    )
+    parser.add_argument(
+        "-c",
+        "--config-path",
+        default=DEFAULT_CONF_PATH,
+        help="File path to the Tron configuration file",
+    )
+    parser.add_argument(
+        "--job-for-stateless-alert",
+        required=True,
+        help="The job name to read timestamp from",
+    )
+    parser.add_argument(
+        "--alert-after",
+        default=DEFAULT_ALERT_AFTER,
+        help="how long (in mins) to wait to alert after the last timestamp of the job",
+    )
+    args = parser.parse_args()
+    args.working_dir = os.path.abspath(args.working_dir)
+    args.config_path = os.path.join(
+        args.working_dir,
+        args.config_path,
+    )
+    return args
+
+
+def read_config(args):
+    return manager.ConfigManager(args.config_path).load().get_master().state_persistence
+
+
+def calc_time_diff(last_run_time, now):
+    def trim_nanoseconds(timestamp):
+        index = timestamp.find('.')
+        if index != -1:
+            timestamp = timestamp[:index]
+        return timestamp
+
+    def _timestamp_to_timeobj(timestamp):
+        return time.strptime(timestamp, '%Y-%m-%d %H:%M:%S')
+
+    time_diff = (
+        now -
+        time.mktime(_timestamp_to_timeobj(trim_nanoseconds(str(last_run_time))))
+    )
+    # secs to mins
+    return time_diff / 60
```

Comment:
What if this function was just called `calculate_seconds_since_last_run_time` ?
Then it would be easier to understand how this function relates to the rest of the code in `main`